treeherder/config/settings.py

```diff
@@ -153,6 +153,7 @@ def server_supports_tls(url):
     'treeherder.autoclassify',
     'treeherder.credentials',
     'treeherder.seta',
+    'django_filters'
```

Comment:
I'd put this above the Treeherder entries and below the other custom Django packages (eg perhaps between corsheaders and graphene_django). Also in Python trailing commas are allowed in dicts/lists/... and are often preferred, since it reduces churn when adding an extra line to the end :-)